bookingsync-engine.gemspec

```diff
@@ -17,6 +17,8 @@ Gem::Specification.new do |s|
 
   s.add_dependency "rails", ">= 5.0.0"
   s.add_dependency "omniauth-bookingsync", '>= 0.5.0'
+  s.add_dependency "omniauth-rails_csrf_protection", "~> 1.0"
+  s.add_dependency "repost"
```

Comment:
do you think it worth to specify the version here?

Reply:
makes sense